File: src/main/java/com/adven/concordion/extensions/exam/commands/ScrollToTopCommand.java
package com.adven.concordion.extensions.exam.commands;

import com.adven.concordion.extensions.exam.html.Html;
import org.concordion.api.CommandCall;
import org.concordion.api.Evaluator;
import org.concordion.api.ResultRecorder;

public class ScrollToTopCommand extends ExamCommand {
    public ScrollToTopCommand(String name, String tag) {
        super(name, tag);
    }

    @Override
    public void setUp(CommandCall cmd, Evaluator evaluator, ResultRecorder resultRecorder) {
        Html el = new Html(cmd.getElement());
        el.childs(Html.Companion.button("").attr("id", "btnToTop").attr("onclick", "topFunction()").childs(
                Html.Companion.italic("").css("fa fa-arrow-up fa-3x")
        ));
    }
}

File: src/main/java/com/adven/concordion/extensions/exam/commands/ScrollToTopCommand.kt
package com.adven.concordion.extensions.exam.commands

import com.adven.concordion.extensions.exam.html.Html
import com.adven.concordion.extensions.exam.html.button
import com.adven.concordion.extensions.exam.html.italic
import org.concordion.api.CommandCall
import org.concordion.api.Evaluator
import org.concordion.api.ResultRecorder

class ScrollToTopCommand(name: String, tag: String) : ExamCommand(name, tag) {

    override fun setUp(cmd: CommandCall?, evaluator: Evaluator?, resultRecorder: ResultRecorder?) {
        Html(cmd!!.element)(
            button("", "id" to "btnToTop", "onclick" to "topFunction()")(
                italic("").css("fa fa-arrow-up fa-3x")
            )
        )
    }
}
